### DISCUSSION TITLE
Stop always recreating default config

### DISCUSSION THREAD

**Discussion by xeruf** [espanso/espanso#1219] [Feature requests and Ideas]
After tracking my espanso config with git I had to remove it to carry out a merge. Unfortunately, espanso kept _instantly_ recreating it, meaning I had to stop the service before continuing.

Maybe it could be configured to only recreate it when started rather than on reload, since this was quite an annoyance.

**Comment by federico-terzi** [comment]
Hey @xeruf,

I'm sorry you experienced this problem! Indeed, I've been bitten by this behavior as well in the past, so with the new version coming #594 it could be a good time to redesign this flow.

Perhaps, we can create an `espanso init` command that creates it only once the first time the user configures espanso

Cheers :)
